variables.tf

```diff
@@ -0,0 +1,103 @@
+variable "namespace" {
+  type = string
+  default = "kube-prometheus-stack"
+}
+
+variable "chart_version" {
+  type = string
+}
+
+variable "grafana_pv_size" {
+  type = string
+  default = "10Gi"
+}
+
+variable "prometheus_pv_size" {
+  type = string
+  default = "100Gi"
+}
+
+variable "ingress_host" {
+  type = string
+}
+
+variable "basic_auth_username" {
+  type = string
+  default = "admin"
+}
+
+variable "cluster_issuer_name" {
+  type = string
+}
+
+variable "install_adapter" {
+  type = bool
+  default = false
+}
+
+variable "grafana_ingress_enabled" {
+  type = bool
+  default = true
+}
+
+variable "adapter_chart_version" {
+  type = string
+  default = ""
+}
+
+variable "pull_secrets" {
+  type = bool
+}
+
+variable "regcred" {
+  type = string
+}
+
+variable "secret_name" {
+  type = string
+}
+
+variable "storage_class_name" {
+  type = string
+  default = ""
+}
+
+variable "prometheus_resources" {
+  type        = any
+  description = "Prometheus resource"
+  default     = { 
+                  cpu_requests = "500m", 
+                  memory_requests = "512Mi" 
+                }
+}
+
+variable "prometheus_operator_resources" {
+  type        = any
+  description = "PrometheusOperetor resource"
+  default     = { 
+                  cpu_requests = "50m", 
+                  memory_requests = "100Mi" 
+                }
+}
+
+variable "grafana_resources" {
+  type        = any
+  description = "Grafana resource"
+  default     = { cpu_requests = "100", 
```

Comment:
Be consistent in the data structure indentation

Reply:
done